pymatgen/analysis/ferroelectricity/ZVAL.yaml

```diff
@@ -0,0 +1,305 @@
+VASP_PBE_ZVAL:
```

Comment:
I've been using this yaml with databases created by atomate workflows that by default do not store the ZVAL for POTCARs but just the pseudopotential type and name. However, I recognize that this is a specific use case issue (and I could modify the workflow to store this information). 

I think this yaml is helpful but not necessary. It does demonstrate what information is needed from pseudopotential files to correctly calculate the ionic polarization. These dictionaries are mildly inconvenient to make if one does not readily have the POTCAR used on hand (or a PSP folder on their local machine) but only the type and name of the pseudopotential used.

Perhaps it would be better to have Outcar extract the ZVALs if lcalcpol = True (modifying __init__ and as_dict). That would solve multiple use cases. Should I go ahead and add the Outcar property if lcalcpol = True and delete the yaml and associated lines in polarization.py?